posts/inside-rust/2020-08-12-changes-to-x-py-defaults.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Changes to x.py defaults"
+author: Joshua Nelson
+team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+Recently, the defaults for [x.py], the tool used to [bootstrap] the Rust compiler from source, changed. If you regularly contribute to Rust, this might affect your workflow.
```

Comment:
Same here if you want to change it (the link at the bottom would also have be to changed).

```suggestion
Recently, the defaults for [`x.py`], the tool used to [bootstrap] the Rust compiler from source, changed. If you regularly contribute to Rust, this might affect your workflow.
```